lib/ex_webrtc/ice/fly_ip_filter.ex

```diff
@@ -0,0 +1,22 @@
+defmodule ExWebRTC.ICE.FlyIpFilter do
+  @moduledoc """
+  ICE IP filter for Fly.io deployments.
+
+  This module defines a single function, which filters out IP addresses,
+  which ICE Agent will use as its host candidates.
```

Comment:
1. "filters out" to me sounds like it's rejecting these addresses rather than accepting them
2. I feel like adding a bit more info about the function could be helpful

```suggestion
  This module defines a single function, which filters IP addresses,
  which ICE Agent will use as its host candidates.
  It accepts only the IPv4 address that `fly-global-services` resolves to.
```